### PR TITLE
chore: remove web3x

### PR DESCRIPTION
**Pull request by menduz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,22 +29,17 @@
     "trailingComma": "none"
   },
   "dependencies": {
-    "bn.js": "5.2.0",
-    "dcl-crypto": "2.3.0",
-    "web3x": "4.0.6"
+    "@dcl/eslint-config": "^1.0.3",
```
Comment (marianogoldman) [inline package.json:32]: Should this be in `devDependencies`?

File: src/utils.ts [hunk 2]
```diff
@@ -1,69 +1,75 @@
...
+    )
+
+  try {
+    const contractAddress = catalystDeployments[networkId]
+
+    return (await new ContractFactory(rm, catalystAbiItems).at(
+      contractAddress
+    )) as any as CatalystContract
+  } catch (error) {
+    return undefined
```
Comment (marianogoldman) [inline src/utils.ts:73]: No need for logging here? Is it the caller's responsibility?

File: test/Catalyst.spec.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import { HTTPProvider } from 'eth-connect'
+import { catalystRegistryForProvider, nameDenylistForProvider, poiListForProvider } from '../src'
+import 'isomorphic-fetch'
+
+
+describe('server list', () => {
+  it('mainnet', async () => {
+    const provider = new HTTPProvider('https://rpc.decentraland.org/mainnet')
```
Comment (marianogoldman) [inline test/Catalyst.spec.ts:8]: Are we making this actual calls in the tests? Wouldn't it be better to mock this?

Comment (menduz) [reply test/Catalyst.spec.ts:8]: We could, but the integration test should be good enough for this case. The libraries used to create the contracts are already unit-tested, we don't need to test them again with mocks, instead, this is more or less an integration test.

File: src/utils.ts (hunk 2 above)
Comment (menduz) [reply src/utils.ts:73]: oops

Comment (menduz) [reply src/utils.ts:73]: I decided that since we are breaking compatibility, bubbling up the error (removing catch) is the best.

**Pull request merged**